core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/dot-experiments-configuration.component.spec.ts

```diff
@@ -102,7 +118,9 @@ describe('DotExperimentsConfigurationComponent', () => {
                 provide: DotMessageService,
                 useValue: messageServiceMock
             },
-
+            { provide: CoreWebService, useClass: CoreWebServiceMock },
+            { provide: DotCurrentUserService, useClass: DotCurrentUserServiceMock },
+            mockProvider(LoggerService),
```

Comment:
Can't we use `mockProvider` to mock `CoreWebService`?

Reply:
tried, but will check again before merge it

Reply:
Let's try mock the complete `Component` if not work the mockProvider